src/ocean/time/Time.d

```diff
@@ -62,7 +62,12 @@ version (unittest)
 struct TimeSpan
 {
         // this is the only member of the struct.
-        package long ticks_;
+        package Duration impl;
```

Comment:
Minor/bikesheddy: something like `duration_` would be a more informative name.

I guess you want to stress the fact that you're reworking the struct as just a thin wrapper around the true implementation, but you know that anyway, so you might as well use a field name that conveys what the field actually is.